contracts/Cargo.toml

```diff
@@ -27,7 +27,7 @@ bin = [
 ]
 
 [dependencies]
-ethcontract = { version = "0.8", default-features = false }
+ethcontract = { git = "https://github.com/gnosis/ethcontract-rs/", rev = "58f82895256189de4ea6efc615ed94f970fc7b0c",  default-features = false }
```

Comment:
Since we maintain ethcontract, should we do a release nowish (@nlordell)?

Reply:
Yes, I imagine we release a new version first. Last time @e00E did the same procedure where he created the PR in services with a git revision pending the `ethcontract` version bump just to get a head start in the review here.